net/shadowsocks/src/opnsense/mvc/app/models/OPNsense/Shadowsocks/General.xml

```diff
@@ -31,5 +30,21 @@
             <default>password</default>
             <Required>N</Required>
         </password>
+        <cipher type="OptionField">
+            <default>AES-256-CFB</default>
+            <Required>Y</Required>
+            <OptionValues>
+                <aes-256-cfb>AES-256-CFB</aes-256-cfb>
+                <aes-256-gcm>AES-256-GCM</aes-256-gcm>
+                <aes-256-ctr>AES-256-CTR</aes-256-ctr>
+                <aes-192-cfb>AES-192-CFB</aes-192-cfb>
+                <aes-192-gcm>AES-192-GCM</aes-192-gcm>
+                <aes-192-ctr>AES-192-CTR</aes-192-ctr>
+                <aes-128-cfb>AES-128-CFB</aes-128-cfb>
+                <aes-128-gcm>AES-128-GCM</aes-128-gcm>
+                <aes-128-ctr>AES-128-CTR</aes-128-ctr>
+                <chacha20-ietf-poly1305>CHACHA20-IETF-POLY1305</chacha20-ietf-poly1305>
```

Comment:
Sorry, this is different :D 

    ChaCha20-IETF-Poly1305

As per https://tools.ietf.org/html/rfc7539